keyboards/hnah40/keymaps/default/keymap.c

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2019 HnahKB
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines the keycodes used by our macros in process_record_user
+enum hnah_layers{
+  _QWERTY,
+  _LOWER,
+  _RAISE
+};
+
+enum hnah_keycodes {
+  QWERTY,
+  LOWER,
+  RAISE 
+};
+
+#define LOWER MO(_LOWER)
```

Comment:
these fines probably mess with the keycodes above. As you're not using the keycodes you could consider removing them.